numberformat_proposed.html

```diff
@@ -402,7 +420,7 @@ <h1>ToRawPrecision( _x_, _minPrecision_, _maxPrecision_ )</h1>
             1. Decrease _cut_ by 1.
           1. If the last character of _m_ is `"."`, then
             1. Remove the last character from _m_.
-        1. Return _m_.
+        1. Return the tuple [_m_, _xFinal_].
```

Comment:
We don't tend to use "tuples" in the JavaScript specification. More typical would be to use Records, such as, `Return the Record { [[Xyz]]: _m_, [[Abc]] _xFinal_ }.`

Reply:
Excellent; thanks!